src/Avalonia.Controls/TopLevel.cs

```diff
@@ -84,6 +91,7 @@ public abstract class TopLevel : ContentControl,
         static TopLevel()
         {
             AffectsMeasure<TopLevel>(ClientSizeProperty);
+            AffectsMeasure<TopLevel>(TotalSizeProperty);
```

Comment:
`TotalSize` isn't settable so not sure why it needs to be flagged as `AffectsMeasure`?